security-policies/bundle/compliance/main.rego

```diff
@@ -7,6 +7,8 @@ import data.compliance.lib.common
 # output is findings
 resource = input.resource
 
+# METADATA
```

Comment:
Sure! I think the "Rationale" section of the rule docs explain it best: https://docs.styra.com/regal/rules/idiomatic/no-defined-entrypoint

It might not be needed here, but it also doesn't hurt, and documenting entrypoints is a pretty good practice.